src/components/ContentMapping/ContentMapping.ts

```diff
@@ -1,41 +1,48 @@
-import React from 'react';
+import React from "react";
 import { PlainText } from "../ContentWidgets/PlainText/PlainText";
 import { PlainTextEditor } from "../ContentWidgets/PlainText/PlainTextEditor";
-import { Stub, StubEditor } from '../ContentWidgets/_Stub/Stub';
-import { ContentSingularData } from '../_data/ContentSingularData';
-import { ExampleImage } from '../ContentWidgets/ExampleImage/ExampleImage';
-import { ExampleImageEditor } from '../ContentWidgets/ExampleImage/ExampleImageEditor';
+import { Stub, StubEditor } from "../ContentWidgets/_Stub/Stub";
+import { ContentSingularData } from "../_data/ContentSingularData";
+import { ExampleImage } from "../ContentWidgets/ExampleImage/ExampleImage";
+import { ExampleImageEditor } from "../ContentWidgets/ExampleImage/ExampleImageEditor";
+import { ToDoList } from "../ContentWidgets/ToDoList/ToDoList";
+import { ToDoListEditor } from "../ContentWidgets/ToDoList/ToDoListEditor";
 
 export type WidgetEditorProps = {
-    originalContent: ContentSingularData,
-    editedContent: ContentSingularData,
-    setEditedContentOnChange: (keyToChange: string, valueToChange: any) => void
-}
+  originalContent: ContentSingularData;
```

Comment:
You should keep tab spaces as 4 and these should also be commas, not semicolons (doesn't matter to the compiler but best stay consistent)

Reply:
If you are using vscode, there should be a `.vscode` file included in this repo. It should automatically override your formatting settings...